packages/axios-extension/src/base/odata-request-error.ts

```diff
@@ -1,3 +1,15 @@
+import type { AxiosError } from 'axios';
+
+/**
+ * Casts an unknown error to an AxiosError.
+ *
+ * @param e unknown error
+ * @returns exception casted to AxiosError if it is one
+ */
+export function isAxiosError(e: unknown): e is AxiosError {
+    return typeof e === 'object' && e !== null && 'isAxiosError' in e;
+}
+
```

Comment:
It's not related to this PR specifically, so I'm not sure it makes sense to change it here (on the other hand, it's a small change).

It's a good idea to set the value of `this.name` in the error class's constructor: https://github.com/SAP/open-ux-tools/blob/da9f6c6d50ee2bf39ac01e3b84a6a9003a79b310/packages/axios-extension/src/base/odata-request-error.ts#L43-L47
Something like this:
```typescript
        this.name = this.constructor.name;
```
Doc: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/name

Reply:
Done https://github.com/SAP/open-ux-tools/pull/797/commits/991e4ef5c8d7dd7433b43a432b8e6fc908699e53